src/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "description": "WebExtension for Motrix download manager",
+  "manifest_version": 2,
+  "name": "Moxtrix WebExtension",
+  "version": "2.0.0",
+  "homepage_url": "https://github.com/gautamkrishnar/motrix-chrome-extension/",
+  "icons": {
+    "16": "assets/images/16.png",
+    "32": "assets/images/32.png",
+    "48": "assets/images/48.png",
+    "128": "assets/images/128.png"
+  },
+  "permissions": [
+    "downloads",
+    "storage",
+    "notifications"
+  ],
+  "browser_action": {
+    "browser_style": true,
+    "default_icon": {
+      "16": "assets/images/16.png",
+      "32": "assets/images/32.png",
+      "48": "assets/images/48.png",
+      "128": "assets/images/128.png"
+    },
+    "default_title": "Motrix WebExtension",
+    "default_popup": "popup.html"
+  },
+  "background": {
+    "scripts": [
+      "browser-polyfill.min.js",
+      "background.js"
+    ],
+    "persistent": true
+  },
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "gautamkrishnar@github.com",
```

Comment:
@gautamkrishnar Replace this with a permanent ID so that I can test the extension in production environment.

Reply:
@149segolte addon id is optional: https://extensionworkshop.com/documentation/develop/extensions-and-the-add-on-id/ lets keep it as `r.gautamkrishna@gmail.com`

Reply:
@gautamkrishnar [We require explicit addon ID to use storage.sync API](https://extensionworkshop.com/documentation/develop/extensions-and-the-add-on-id/#when-do-you-need-an-add-on-id:~:text=Some%20WebExtension%20APIs%20use%20the%20add%2Don%20ID%20and%20expect%20it%20to%20be%20the%20same%20from%20one%20browser%20session%20to%20the%20next.%20If%20you%20use%20these%20APIs%2C,runtime.onConnectExternal)

I will update the ID.